src/stores/video.ts

```diff
@@ -642,27 +760,32 @@ export const useVideoStore = defineStore('video', () => {
   )
 
   return {
-    availableIceIps,
     allowedIceIps,
-    namesAvailableStreams,
-    videoStoringDB,
-    tempVideoChunksDB,
-    discardFilesFromVideoDB,
-    downloadFilesFromVideoDB,
-    clearTemporaryVideoDB,
-    downloadTempVideoDB,
-    unprocessedVideos,
-    keysAllUnprocessedVideos,
-    keysFailedUnprocessedVideos,
     areThereVideosProcessing,
-    processUnprocessedVideos,
+    availableIceIps,
+    clearTemporaryVideoDB,
+    currentFileProgress,
+    discardProcessedVideosFromDB,
     discardUnprocessedVideos,
-    temporaryVideoDBSize,
-    videoStorageFileSize,
+    discardUnprocessedVideosFromDB,
+    downloadFilesFromVideoDB,
+    downloadTempVideo,
+    downloadTempVideoDB,
     getMediaStream,
     getStreamData,
     isRecording,
-    stopRecording,
+    keysAllUnprocessedVideos,
+    keysFailedUnprocessedVideos,
+    namesAvailableStreams,
+    overallProgress,
+    processAllUnprocessedVideos,
+    processVideoChunksAndTelemetry,
     startRecording,
+    stopRecording,
+    temporaryVideoDBSize,
+    tempVideoChunksDB,
+    unprocessedVideos,
+    videoStorageFileSize,
+    videoStoringDB,
```

Comment:
I think there are changes here being done by a formatting tool, right?

Could those be separated from this commit, so it's more clear what is being changed?